package/kernel/linux/modules/netsupport.mk

```
@@ -1476,6 +1476,21 @@ endef
 
 $(eval $(call KernelPackage,xdp-sockets-diag))
 
+define KernelPackage/inet-mptcp-diag
+  SUBMENU:=$(NETWORK_SUPPORT_MENU)
+  TITLE:=INET diag support for MultiPath TCP
+  DEPENDS:= @KERNEL_MPTCP @KERNEL_MPTCP_IPV6 +kmod-inet-diag
+  KCONFIG:= CONFIG_INET_MPTCP_DIAG
+  FILES:= $(LINUX_DIR)/net/mptcp/mptcp_diag.ko
+  AUTOLOAD:=$(call AutoProbe,mptcp_diag)
+endef
+
+define KernelPackage/inet-mptcp-diag/description
+Support for INET (MultiPath TCP) socket monitoring interface used by
```

This isn't an explanation of INET, rather an extension for it:

```suggestion
Support for INET MultiPath TCP socket monitoring interface used by
```

---

Done.

---

> Done.

Hmmm... It doesn't seem so?

---

My mistake, sorry. Fixed.